src/lib/lifeCycle.ts

```diff
@@ -57,35 +60,85 @@ import { getIsInteractive } from "./helpers/getIsInteractive.js";
 import { stringifyUnknown } from "./helpers/jsonStringify.js";
 import { getLatestVersionOfNPMDependency } from "./npm.js";
 import {
+  ensureUserFluenceDir,
   projectRootDir,
   recursivelyFindProjectRootDir,
   setProjectRootDir,
 } from "./paths.js";
-import { confirm } from "./prompt.js";
+import { confirm, list } from "./prompt.js";
+
+const NO_DOCS_IN_CONFIGS =
+  "Only include a comment with a link to the online docs";
+
+const DOCS_IN_CONFIGS = "Include commented-out examples and docs";
+
+const DOCS_IN_CONFIGS_OPTIONS = [NO_DOCS_IN_CONFIGS, DOCS_IN_CONFIGS] as const;
+
+const promptUserAboutDocsInConfigs = async (): Promise<boolean> => {
+  if (!isInteractive) {
+    return false;
+  }
+
+  const docsInConfigsOption = await list({
+    message:
+      "Select what comments you want to see in the configs generated by the CLI",
+    oneChoiceMessage: () => {
+      return commandObj.error("Unreachable");
+    },
+    onNoChoices: () => {
+      return commandObj.error("Unreachable");
+    },
+    options: [...DOCS_IN_CONFIGS_OPTIONS],
+  });
+
+  commandObj.logToStderr(
+    `If you change your mind later, modify "docsInConfigs" property in ${join(
```

Comment:
is that a global-only setting?

Reply:
this is per-user. I think it's fare. If cli user wants it to be different - he can go ahead and change it